timemachine/cpp/src/kernels/k_fixed_point.cuh

```diff
@@ -67,3 +67,30 @@ template <typename RealType> unsigned long long __device__ __forceinline__ FLOAT
 template <typename RealType> unsigned long long __device__ __forceinline__ FLOAT_TO_FIXED_BONDED(RealType v) {
     return static_cast<unsigned long long>(real_to_int64(v * FIXED_EXPONENT));
 }
+
+/* FLOAT_TO_FIXED_ENERGY converts floating point energies into fixed point. For infinite values or values beyond LLONG_MAX/LLONG_MIN
+* the value will be capped. This is accumulated into __int128 which handles the over and underflows, allowing us to be able to account
```

Comment:
explicitly state `capped to LLONG_MAX/LLONG_MIN`

Reply:
Made more explicit in 742757945f3dc64bf5a38879e908507e86bad7a7